tock/hours/views.py

```diff
@@ -802,6 +802,7 @@ def get_context_data(self, **kwargs):
         context = super(
             ReportingPeriodUserDetailView, self).get_context_data(**kwargs)
         context['user_billable_hours'] = timecard.billable_hours
+        context['user_weekly_allocation'] = timecard.total_weekly_allocation
```

Comment:
Definitely out of scope for this work, but noting that the `total_weekly_allocation` name is becoming increasingly imprecise as we continue the utilization work. Something like `billable_allocation` is probably more descriptive (as hinted by the new "Billable Allocation" table header) and consistent with the `billable_hours`/`non_billable_hours` nomenclature that predates weekly billing.

Reply:
I agree, the problem began when I added a filter so that field would only be billable, but the name didn't change.  Let's make a ticket!